rviz_default_plugins/include/rviz_default_plugins/displays/tf/tf_display.hpp

```diff
@@ -118,6 +119,8 @@ private Q_SLOTS:
   FrameInfo * createFrame(const std::string & frame);
   void updateFrame(FrameInfo * frame);
   void deleteFrame(FrameInfo * frame, bool delete_properties);
+  typedef std::map<std::string, FrameInfo *> M_FrameInfo;
+  TFDisplay::M_FrameInfo::iterator deleteFrame(M_FrameInfo::iterator it, bool delete_properties);
```

Comment:
Do we actually need the `TFDisplay` here, as this is the class we are currently declaring?